app/src/main/java/com/example/quizacademytask/FlashcardStackFragment.kt

```diff
@@ -77,7 +79,7 @@ class FlashcardStackFragment : Fragment() {
             val frag = CardFragment()
```

Comment:
no need for SlidePagerAdapter to be an inner class. Also extract it to a separate file